templates/conf/zoo.cfg.erb

```diff
@@ -66,3 +66,7 @@ autopurge.purgeInterval=<%= scope.lookupvar('zookeeper::config::purge_interval')
 
 # Maximum allowed connections
 maxClientCnxns=<%= scope.lookupvar('zookeeper::config::max_allowed_connections') %>
+
+# Listen for connections from peers on all available IPs
+# Default is false; but, at Yelp, via Puppet the default is set to true
+quorumListenOnAllIPs=<%= scope.lookupvar('zookeeper::config::quorum_listen_on_all_ips') %>
```

Comment:
This will need to be passed from init and be a class variable in it.

Otherwise, we will not be able to specify this at hiera level for zookeeper_cluster module.

Also, spec files need to be updated for this.

Reply:
@ronin13 Made the necessary changes; completely forgot about this :-(